packages/braid-design-system/src/lib/components/icons/IconSentiment/IconSentiment.tsx

```diff
@@ -0,0 +1,28 @@
+import type React from 'react';
+import { Box } from '../../Box/Box';
+import type { UseIconProps } from '../../../hooks/useIcon';
+import useIcon from '../../../hooks/useIcon';
+import { IconSentimentSvg } from './IconSentimentSvg';
+import { IconSentimentNegativeSvg } from './IconSentimentNegativeSvg';
+import { IconSentimentPositiveSvg } from './IconSentimentPositiveSvg';
+
+type Feeling = 'positive' | 'negative' | 'neutral';
+
+export type IconSentimentProps = UseIconProps & {
+  feeling?: Feeling;
+};
+
+const feelingToIcon: Record<Feeling, React.ComponentType> = {
+  positive: IconSentimentPositiveSvg,
+  negative: IconSentimentNegativeSvg,
+  neutral: IconSentimentSvg,
+};
+
+export const IconSentiment = ({
+  feeling = 'neutral',
+  ...props
+}: IconSentimentProps) => {
+  const iconProps = useIcon(props);
+
+  return <Box component={feelingToIcon[feeling]} {...iconProps} />;
```

Comment:
Should we fall back to a sentiment if an invalid `feeling` is provided? Feel like it's worth it to prevent an unexpected empty icon.

```suggestion
  return <Box component={feelingToIcon[feeling] || IconSentimentSvg} {...iconProps} />;
```

[`IconChevron`](https://github.com/seek-oss/braid-design-system/blob/fb60dfcbfba5699ddd4db0c78ef21092f448a370/packages/braid-design-system/src/lib/components/icons/IconChevron/IconChevron.tsx) is the only other icon that has a non-boolean prop that affects the icon appearance, but it implements that using CSS, so an invalid input will just result in the `down` appearance.